gen/templates/profile_parcels/model.go

```diff
@@ -45,7 +45,11 @@ type {{camelCase .Name}} struct {
 {{- if isNestedListSet .}}
 	{{toGoName .TfName}} []{{$name}}{{toGoName .TfName}} `tfsdk:"{{.TfName}}"`
 {{- else}}
+	{{- if eq .Type "StringInt64"}}
```

Comment:
I think this can be come an "else if" instead of a nested "if":
```go
{{- if isNestedListSet .}}
	{{toGoName .TfName}} []{{$name}}{{toGoName .TfName}} `tfsdk:"{{.TfName}}"`
{{-else if eq .Type "StringInt64"}}
	{{toGoName .TfName}} types.String `tfsdk:"{{.TfName}}"`
{{- else}}
	{{toGoName .TfName}} types.{{.Type}} `tfsdk:"{{.TfName}}"`
{{- end}}
```